.gitignore

```diff
@@ -20,8 +20,8 @@
 *.byebug_history
 
 # Ignore application configuration
-/config/env/test.env
-/config/env/development.env
+/config/env/test.example.env
```

Comment:
I do remember reading this part, probably just committed some mistake with  the .gitgnorefile